### PR TITLE
ROC-7698: Decommission claimant reject organisation plan emails

### PR DESCRIPTION
**Pull request by daveoh**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-7698

### Change description ###
Decommission claimant reject organisation plan emails

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

### CODE REVIEW

File: definition/data/sheets/CaseEvent.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
...
   {"CaseTypeID": "MoneyClaimCase", "ID": "AgreementSignedByClaimant", "Name": "Claimant signed SA", "Description": "Claimant Signs Settlement", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "RejectOrganisationPaymentPlan", "Name": "Plan proposed for Org", "Description": "New plan proposed for organisation", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState": "open", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "RejectOrganisationPaymentPlan", "Name": "Plan proposed for Org", "Description": "New plan proposed for organisation", "DisplayOrder": 1, "PreConditionState(s)": "open", "PostConditionState(s)": "open;judgmentDecideAmount", "SecurityClassification": "Public", "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CLAIM_STORE_BASE_URL}/cases/callbacks/about-to-submit", "RetriesTimeoutURLAboutToSubmitEvent": "5,10,15"},
```
Comment (ckalista) [inline definition/data/sheets/CaseEvent.json]: I think it is preferable to disable retries for now at least until we identify a clear need/benefit. Just as always concerned about side effects.

File: definition/data/sheets/AuthorisationCaseState.json [hunk 2]
```diff
@@ -47,5 +47,8 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "awaitingCitizenPayment", "UserRole": "citizen", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "readyForJudgeDirections", "UserRole": "caseworker-cmc-judge", "CRUD": "CRUD"},
   {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "readyForJudgeDirections", "UserRole": "caseworker-cmc", "CRUD": "CRUD"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "readyForJudgeDirections", "UserRole": "citizen", "CRUD": "CRUD"}
+  {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "readyForJudgeDirections", "UserRole": "citizen", "CRUD": "CRUD"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseStateID": "judgmentDecideAmount", "UserRole": "caseworker-cmc-legaladvisor", "CRUD": "CUD"},
```
Comment (ckalista) [inline definition/data/sheets/AuthorisationCaseState.json]: Don't forget this new state needs to be 'hidden' in this definition.

Comment (daveoh) [reply definition/data/sheets/AuthorisationCaseState.json]: Removing read access hides this state

File: definition/data/sheets/CaseEvent.json (hunk 1 above)
Comment (daveoh) [reply definition/data/sheets/CaseEvent.json]: Done

File: definition/data/sheets/AuthorisationCaseState.json (hunk 2 above)
Comment (ckalista) [reply definition/data/sheets/AuthorisationCaseState.json:51]: 👍

**Pull request merged**